specification/trafficmanager/resource-manager/Microsoft.Network/stable/2020-12-01/trafficmanager.json

```diff
@@ -979,6 +979,16 @@
           "format": "int64",
           "description": "The minimum number of endpoints that must be available in the child profile in order for the parent profile to be considered available. Only applicable to endpoint of type 'NestedEndpoints'."
         },
+        "minChildEndpointsIPv4": {
+          "type": "integer",
+          "format": "int64",
+          "description": "The minimum number of endpoints that must be available in the child profile in order for the parent profile to be considered available. Only applicable to endpoint of type 'NestedEndpoints'."
```

Comment:
not a blocker but something to consider is adding more info in your description for example limits, can I enter 5000? Or what happens if I enter 0? Also if you have a limit, then you can use min and max value like [here](https://swagger.io/docs/specification/data-models/data-types/#numbers)

Reply:
Will add more helpful information in next revision. Thank you!